rotkehlchen/exchanges/binance.py

```diff
@@ -719,6 +832,11 @@ def query_lending_interests_history(
                     queried_ranges=[(start_ts, end_ts)],
                 )
 
+        if events_failed_to_save > 0:
+            self.msg_aggregator.add_error(
+                f'Failed to save {events_failed_to_save} binance lending history events in the DB. Check logs.',  # noqa: E501
+            )
```

Comment:
Remove this logic. We don't need to spam user with this. If they are missing events they can go and check the logs.

```suggestion
```